<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientManagerForm</class>
 <widget class="QWidget" name="ClientManagerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>412</width>
     <height>307</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QListWidget" name="listWidget"/>
   <widget class="QToolBox" name="toolBox">
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="inputPage">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>202</width>
       <height>249</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Input</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QFormLayout" name="formLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="labelNum">
          <property name="text">
           <string>Num</string>
          </property>
          <property name="buddy">
           <cstring>lineEditNum</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="labelName">
          <property name="text">
           <string>&amp;Name</string>
          </property>
          <property name="buddy">
           <cstring>lineEditName</cstring>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="labelAddress">
          <property name="text">
           <string>&amp;Address</string>
          </property>
          <property name="buddy">
           <cstring>lineEditAddress</cstring>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="labelPhoneNumber">
          <property name="text">
           <string>&amp;Phone Number</string>
          </property>
          <property name="buddy">
           <cstring>lineEditPhoneNumber</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEditNum">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="lineEditName"/>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="lineEditPhoneNumber"/>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="lineEditAddress"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelPW">
          <property name="text">
           <string>Password</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEditPW"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelID">
          <property name="text">
           <string>ID</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEditID"/>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>&amp;Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonModify">
          <property name="text">
           <string>&amp;Modify</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="SearchPage">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>202</width>
       <height>249</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Search</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QListWidget" name="listWidgetSearch"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>&amp;ID</string>
          </property>
          <property name="buddy">
           <cstring>lineEditSearch</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditSearch"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>Search</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
